hermes-console-vue/src/store/app-notifications/useAppNotifications.ts

```diff
@@ -0,0 +1,31 @@
+import { defineStore } from 'pinia';
+import { v4 as generateUUID } from 'uuid';
+import type {
+  Notification,
+  NotificationConfig,
+  NotificationsState,
+} from '@/store/app-notifications/types';
+
+export const useNotificationsStore = defineStore('notifications', {
+  state: (): NotificationsState => {
+    return {
+      notifications: [],
+    };
+  },
+  actions: {
+    async dispatchNotification({ title, text, type }: NotificationConfig) {
+      const notification: Notification = {
+        id: generateUUID(),
```

Comment:
Minor: what about using window.crypto.randomUUID() instead? no need for another dependency

Reply:
But then notification store would not work on http domains